dev-tools/xtask/src/virtual_hardware.rs

```diff
@@ -34,6 +34,10 @@ enum Commands {
         #[clap(long)]
         gateway_ip: Option<String>,
 
+        /// The configured mode for softnpu
+        #[clap(long, env, default_value = "zone")]
+        softnpu_mode: String,
```

Comment:
Can this be an arbitrary string? If it's only one of a small number of values, it'd be better to use `ValueEnum`.

Reply:
I believe it's determined by the caller, and may have uses outside this xtask. If for some reason some using software changed it, I wouldn't want deployment to break. However, I'm also not sure it's that big of a deal to require updating this script.